core/amber/src/main/scala/edu/uci/ics/texera/workflow/common/tuple/schema/AttributeTypeUtils.scala

```diff
@@ -146,24 +145,38 @@ object AttributeTypeUtils extends Serializable {
     val parseError = new AttributeTypeException(
       s"not able to parse type ${fieldValue.getClass} to Timestamp: ${fieldValue.toString}"
     )
-    val datetimeISOFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'")
-    val dateFormat = new SimpleDateFormat("yyyy-MM-dd")
+    // Extensive list of potential date formats
+    val dateFormats = List(
```

Comment:
two questions. 
1. can we add examples
2. can we use a library to do so?

Reply:
I decided to use dateparser lib in java.